src/app/helpers/tag-manager.js

```diff
@@ -27,59 +28,14 @@ window.dataLayer = window.dataLayer || [];
     }
 })(window, document, 'script', 'dataLayer', tagManagerID);
 
-/*
```

Comment:
i moved this deleted logic into the analytics helper so it could be shared with rex.